### PR TITLE
quartus-prime-lite : 20.1.1.720 -> 22.1std.0.915

### PR DESCRIPTION
**Pull request by reixn**
###### Description of changes

Update quartus to 22.1.
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by bjornfor** [comment]
Since this diff is a little more than changed `version` and `sha256`, can you please say something about it in the commit message?

**Comment by bjornfor** [comment]
> Since this diff is a little more than changed `version` and `sha256`, can you please say something about it in the commit message?

Or perhaps the commit can be split in two (refactor + bump)? (As long as it doesn't break git bisect.)

**Comment by reixn** [comment]
Added some description in commit message

**Comment by bjornfor** [comment]
```
$ gh pr checkout 204401
$ nix-build -A quartus-prime-lite
...
error: hash mismatch in fixed-output derivation '/nix/store/wmbh6x1f1cwakmhay8z5wm7608yx3ywd-QuestaSetup-22.1std.0.915-linux.run.drv':
         specified: sha256-HRvJpQNIEsIWA4w4Daf6pmWb7BqHUnR1hdckFzuBFKs=
            got:    sha256-3AU6n0XbFnTAhaUgaSJPFPYIHCY2hQOhbvIJx4yfLtM=
```

Any ideas why that happened?

**Comment by reixn** [comment]
I can't reproduce the same problem. Can you check if result file matches the sha1 digest on Intel's website?

**Comment by bjornfor** [comment]
The URL it fetches (`https://cdrdv2.intel.com/v1/dl/getContent/757261/757277?filename=QuartusLiteSetup-22.1std.0.915-linux.run`) is a HTML page with a license agreement accept / reject options, not the actual file we want.

**Comment by bjornfor** [comment]
I checked on master branch, and that URL (`https://downloads.intel.com/akdlm/software/acdsinst/20.1std.1/720/ib_installers/QuartusLiteSetup-20.1.1.720-linux.run`) works -- it fetches the file instead of HTML license page.

File: pkgs/applications/editors/quartus-prime/default.nix [hunk 1]
```diff
@@ -22,6 +22,8 @@ in buildFHSUserEnv rec {
     xorg.libICE
     xorg.libSM
     zlib
+    dbus
+    libxcrypt
```
Comment (bjornfor) [inline pkgs/applications/editors/quartus-prime/default.nix:26]: libxcrypt is already there in master branch. Does this PR need a rebase, perhaps? (And update the commit message.)

**Comment by reixn** [comment]
Move libxcrypt to multiPkgs.

**Comment by haoxiangliew** [comment]
This update results in the a core dump for quartus.
vsim (QuestaSim) still works fine.

dump

[dump.txt](https://github.com/NixOS/nixpkgs/files/10829061/dump.txt)

**Review by bjornfor — CHANGES_REQUESTED** [review]
From https://github.com/NixOS/nixpkgs/pull/204401#issuecomment-1444569696:

> This update results in a core dump for quartus.
> vsim (aka: QuestaSim) still works fine.

**Comment by haskelious** [comment]
This PR is obsoleted by https://github.com/NixOS/nixpkgs/pull/277610

**Comment by bjornfor** [comment]
> This PR is obsoleted by https://github.com/NixOS/nixpkgs/pull/277610

Yes, closing this one.

**Pull request closed without merging**